crates/json-abi/src/to_sol.rs

```diff
@@ -20,6 +20,9 @@ pub struct ToSolConfig {
     print_constructors: bool,
     enums_as_udvt: bool,
     for_sol_macro: bool,
+    /// Default is false. If set to `true`, any types part of some other interface/library are
+    /// generated in the same interface as the contract and not in a separate library.
+    types_in_interface: bool,
```

Comment:
these docs make sense, but I wonder if we can be a bit more explicit with the field name here, but since this is an advanced setting this seems fine